src/goto-symex/builtin_functions.cpp

```diff
@@ -355,6 +355,64 @@ void goto_symext::symex_printf(const expr2tc &, const expr2tc &rhs)
     cur_state->guard.as_expr(), cur_state->source, fmt.as_string(), args);
 }
 
+void goto_symext::symex_fscanf(
+  const expr2tc &lhs,
+  const std::vector<expr2tc> &op)
+{
+  auto input_stream = op[0]; // stdin?
+  cur_state->rename(input_stream);
+  input_stream->dump();
+
+  /* TODO: Check for Format
```

Comment:
Are we going to keep these comments?

Reply:
Yes, this is not a complete implementation.

Reply:
Could you please open an issue to remind us to implement this TODO?